addons/pos_restaurant/static/src/app/control_buttons/control_buttons.xml

```diff
@@ -41,7 +41,7 @@
             </button>
         </xpath>
         <xpath expr="//button[hasclass('o_fiscal_position_button')]" position="attributes">
-            <attribute name="t-if">!pos.config.takeaway</attribute>
+            <attribute name="t-if">!pos.config.takeaway and pos.cashier._role != 'minimal'</attribute>
```

Comment:
```suggestion
            <attribute name="t-if">!pos.config.takeaway or pos.cashier._role != 'minimal'</attribute>
```
Check this

Reply:
It will `and` only because there will be cases where might one condition fulfill and minimal condition is not even checked